src/gui/properties/propertieswidget.ui

```diff
@@ -788,23 +782,39 @@
                </widget>
               </item>
               <item row="2" column="0">
-               <widget class="QLabel" name="labelHash">
+               <widget class="QLabel" name="labelInfohash1">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
-                 <string>Torrent Hash:</string>
+                 <string>Infohash v1:</string>
```

Comment:
Maybe "Info hash v1" to remain consistent with the GUI?

Reply:
Fixed.